code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -89,28 +89,20 @@
 
 /obj/item/weapon/circuitboard/airlock/Topic(href, href_list)
 	if(..()) return 1 //Its not as though this does ANYTHING
-	if(!Adjacent(usr) || usr.stat || usr.restrained() || (!ishuman(usr) && !isrobot(usr)) || icon_state == "door_electronics_smoked" || installed)
+	if(!Adjacent(usr) || usr.restrained() || (!ishuman(usr) && !isrobot(usr)) || icon_state == "door_electronics_smoked" || installed)
```

Comment:
That's a weird check for restrained(), shouldn't it be under incapacitated() or whatever?